### ISSUE TITLE
7Zip problems on Windows CI builds

### ISSUE THREAD

**Issue by johnthagen** [sfackler/rust-openssl#739]
Errors of the form:

```
Error uncompressing cache item: 7z.exe process has exited with code 2. Check C:\Users\appveyor\AppData\Local\Temp\1\build-cache-logs\0e8a3ad980ec179856012b7eecf4327e99cd44cd.zip.001.log for details.
```

Are causing Windows CI builds to fail.

https://ci.appveyor.com/project/sfackler/rust-openssl/build/1.0.1702/job/vhmb4g1l34wxugwt#L5

**Comment by sfackler** [comment]
It seems like we need to clear the caches out, but it unfortunately doesn't seem like there's a particularly straightforward way to do it :(

**Comment by sfackler** [comment]
Fixed in #767.
